File: app/src/main/java/com/countries/mycountries/model/Worldpopulation.java

package com.countries.mycountries.model;


import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class Worldpopulation {

    public Integer rank;
    public String country;
    public String population;
    public String flag;

    /**
     * No args constructor for use in serialization
     * 
     */
    public Worldpopulation() {
    }

    /**
     * 
     * @param rank
     * @param flag
     * @param population
     * @param country
     */
    public Worldpopulation(Integer rank, String country, String population, String flag) {
        super();
        this.rank = rank;
        this.country = country;
        this.population = population;
        this.flag = flag;
    }

}

File: My Countries-kotlin/app/src/main/java/com/countries/mycountries/model/Worldpopulation.kt
package com.countries.mycountries.model


import org.parceler.Parcel



@Parcel
class Worldpopulation {

     var rank: Int? = null
     var country: String? = null
     var population: String? = null
     var flag: String? = null

    /**
     * No args constructor for use in serialization

     */
    constructor() {}

    /**

     * @param rank
     * *
     * @param flag
     * *
     * @param population
     * *
     * @param country
     */
    constructor(rank: Int?, country: String, population: String, flag: String) : super() {
        this.rank = rank
        this.country = country
        this.population = population
        this.flag = flag
    }

}
